ShopGo/Login/LoginViewController.swift

```diff
@@ -21,4 +21,18 @@ class LoginViewController: UIViewController {
     
     @IBAction func loginButtonAction(_ sender: UIButton) {
     }
+    
+    @IBAction func facebookButton(_ sender: Any) {
+        UIApplication.shared.open(URL(string:"https://www.facebook.com")! as URL , options: [:] , completionHandler: nil)
```

Comment:
create a function for this instead of writing the code again and again

Reply:
fixed